optimize/backend/src/main/java/io/camunda/optimize/rest/security/AbstractSecurityConfigurerAdapter.java

```diff
@@ -51,7 +51,8 @@ protected SecurityFilterChain applyPublicApiOptions(final HttpSecurity http) {
               oauth2resourceServer ->
                   oauth2resourceServer.jwt(
                       jwtConfigurer -> jwtConfigurer.decoder(publicApiJwtDecoder())))
-          .build();
+          // .build();
```

Comment:
I replaced the `.build()` with `.getObject()` here (and in other places) in hopes to stop Optimize from building (and clashing) with the auth module

Downside here is that none of this worked and the error still exists on startup